solutions/community-deploy/cloud/AWS/README.md

```diff
@@ -84,44 +74,19 @@ for ip in $PUBLIC_IP;do rsync -avzrP -e 'sudo ssh -i cortx.pem -o StrictHostKeyC
-ssh -i cortx.pem -o 'StrictHostKeyChecking=no' centos@$AWS_PRIMARY_IP
-git clone https://github.com/Seagate/cortx-re && cd $PWD/cortx-re/solutions/community-deploy
-time bash -x ./build-cortx.sh
-```
-- Save and compress the cortx build images by running following command,
-```
-echo "Wait till the operation is completed..." && docker save $(docker images | sed '1d' | awk '{print $1 ":" $2 }') -o /tmp/cortximages.tar
-```
-- Execute the following command to copy the cortx build images from **primary node to all the worker nodes using private ip address**,
-```
-AWS_WORKER_IP=$(cat /tmp/ip_private.txt | jq '.[1]','.[2]' | tr -d '",[]')
-for worker in $AWS_WORKER_IP;do cd /tmp && rsync -avzrP -e 'sudo ssh -i cortx.pem -o StrictHostKeyChecking=no' /tmp/*.tar /etc/docker/daemon.json centos@$worker:/tmp; done
-```
-- Load the cortx build images to the worker nodes by executing the following command,
-```
-for worker in $AWS_WORKER_IP;do ssh -i cortx.pem -o 'StrictHostKeyChecking=no' centos@$worker 'sudo docker load -i /tmp/cortximages.tar'; done
+export PRIMARY_IP=$(cat ip_public.txt | jq '.[0]'| tr -d '",[]')
+ssh -i cortx.pem -o 'StrictHostKeyChecking=no' centos@$PRIMARY_IP "git clone https://github.com/Seagate/cortx-re && cd $PWD/cortx-re/solutions/community-deploy && time bash -x ./build-cortx.sh"
 ```
 - Clone cortx-re repository from required branch/tag. If you do not provide `-b <branch/tag>`, then it will use default main branch    
   :warning: Tag based build is supported after including tag [2.0.0-879](https://github.com/Seagate/cortx-re/releases/tag/2.0.0-879)
   
-**Note:** If you had cloned cortx-re repo earlier based on above instructions then remove it before following with `branch/tag`
+**Note:** If you had cloned cortx-re repo earlier based on above instructions then remove it before following with `branch/tag` and to use the latest release from https://github.com/Seagate/cortx/releases
 ```
 git clone https://github.com/Seagate/cortx-re -b <branch/tag> && cd $PWD/cortx-re/solutions/community-deploy
 ```
 
-**For example:**
-```
-git clone https://github.com/Seagate/cortx-re -b 2.0.0-879 && cd $PWD/cortx-re/solutions/community-deploy
-```
-- Generate CORTX container images from required branch/tag. If you do not provide `-b <branch/tag>`, then it will use default main branch  
-  :warning: Tag based build is supported after and including tag [2.0.0-879](https://github.com/Seagate/cortx-re/releases/tag/2.0.0-879)
-```
-sudo time bash -x ./build-cortx.sh -b <branch/tag>
-```
-
 ### CORTX Deployment
-- After CORTX build is ready, follow [CORTX Deployment](https://github.com/Seagate/cortx-re/blob/main/solutions/community-deploy/CORTX-Deployment.md) to deploy CORTX on AWS instance.   
+- After CORTX build is ready, follow [CORTX Deployment](https://github.com/Seagate/cortx-re/blob/main/solutions/community-deploy/CORTX-Deployment.md) to deploy CORTX on AWS instances.   
 - Please exclude SELINUX and Hostname setup steps.
```

Comment:
We will need to add AWS-specific steps - 

- To update the hosts file with `private hostnames`
- Update the docker config file to pull images from the local registry configured on the build server.

Reply:
done